resources/tenants.py

```diff
@@ -14,49 +14,57 @@
 # | PUT    | `v1/tenants/:id`     | Updates a single tenant   |
 # | DELETE | `v1/tenants/:id`     | Deletes a single tenant   |
 
+
 class Tenants(Resource):
     @admin_required
     def get(self, tenant_id=None):
         # GET /tenants
         if not tenant_id:
-            return {'tenants': [tenant.json() for tenant in TenantModel.query.all()]}
+            return {"tenants": [tenant.json() for tenant in TenantModel.query.all()]}
 
         # GET /tenants/<tenant_id>
         tenant = TenantModel.find_by_id(tenant_id)
         if not tenant:
-            return {'message': 'Tenant not found'}, 404
+            return {"message": "Tenant not found"}, 404
         return tenant.json()
 
-
     @admin_required
     def post(self):
         tenantEntry = TenantModel.create(schema=TenantSchema, payload=request.json)
 
         returnData = tenantEntry.json()
 
         leaseData = request.json
-        leaseData.update({'tenantID': tenantEntry.id})
+        leaseData.update({"tenantID": tenantEntry.id})
 
-        #if this tenant has a lease
-        if ("dateTimeEnd" in leaseData and "dateTimeStart" in leaseData and "propertyID" in leaseData):
-            LeaseModel.create(
-                schema=LeaseSchema,
-                payload=leaseData
+        # if this tenant has a lease
+        if (
+            "dateTimeEnd" in leaseData
+            and "dateTimeStart" in leaseData
+            and "propertyID" in leaseData
+        ):
```

Comment:
We should probably create an issue to refactor this or do it now. I'd like to remove this completely, but that might not be possible without changing the frontend.

There are two issues here that make it hard to read.
1. The comment
2. the multi-line if statement

I would suggest changing it to something like this instead:
```
if _lease():
...
...


def _lease():
     return (
         "dateTimeEnd" in leaseData
         and "dateTimeStart" in leaseData
         and "propertyID" in leaseData
     )
```

Reply:
That makes sense.  I've refactored it and got it working.  If it still bothers us down the line, we can always refactor again 😆